lblod/ember-rdfa-editor-lblod-plugins
# Update `@lblod/ember-rdfa-editor` peerdependency requirement to 10.7.1

### Overview
This PR contains the following changes:
- Update of  `@lblod/ember-rdfa-editor` peerdependency requirement to 10.7.1
- Addition of `BlockRdfaView` nodeview to dummy pages
- Small adjustment to mandatee-table css to reduce margins of warning message (to counter some styling introduced in version [10.7.0](https://github.com/lblod/ember-rdfa-editor/releases/tag/v10.7.0) of the editor)

##### connected issues and PRs:
Depends on https://github.com/lblod/ember-rdfa-editor/pull/1226

### Setup
None

### How to test/reproduce
- Start the dummy app
- Ensure the `block_rdfa` nodes have the new design on all dummy pages
- Ensure the mandatee table and template comment nodes have the correct margins

### Challenges/uncertainties
This change is of course breaking, as it introduces a peer dependency version increase

### Checks PR readiness
